components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -5,27 +5,28 @@ export default {
   key: "calendly_v2-get-event",
   name: "Get Event",
   description: "Gets information about an Event associated with a URI.",
-  version: "0.1.1",
+  version: "0.1.2",
   type: "action",
   props: {
     calendly_v2: {
       type: "app",
       app: "calendly_v2",
     },
-    event_uuid: {
+    event_url: {
```

Comment:
Should add a `label` for a more user friendly name of the prop -- something like, "Event URL".